integration-tests/virtual-threads/jms-virtual-threads/pom.xml

```diff
@@ -36,7 +36,7 @@
         <dependency>
             <groupId>io.smallrye.reactive</groupId>
             <artifactId>smallrye-reactive-messaging-jms</artifactId>
-            <version>4.8.0</version>
+            <version>4.9.0</version>
```

Comment:
@ozangunalp I wonder if this one should be added to the BOM instead. Probably a good idea to keep things consistent for users.